iothub_client/inc/iothub_message.h

```diff
@@ -356,6 +355,28 @@ MOCKABLE_FUNCTION(, IOTHUB_MESSAGE_RESULT, IoTHubMessage_SetConnectionModuleId,
 */
 MOCKABLE_FUNCTION(, const char*, IoTHubMessage_GetConnectionDeviceId, IOTHUB_MESSAGE_HANDLE, iotHubMessageHandle);
 
+/**
+* @brief   Sets the message creation time in UTC. CAUTION: SDK user should not call it directly, it is for internal use only.
+*
+* @param   iotHubMessageHandle Handle to the message.
+* @param   messageCreationTimeUtc Pointer to the message creation time
+*
```

Comment:
So as I can see in the PR this is not implemented for AMQP transport.
That should be documented in these functions as a remark.

Reply:
should I?

Reply:
That would be good